packages/common/src/ui/layout/tables/columns/NameAndColorColumn.tsx

```diff
@@ -17,55 +17,21 @@ export const getNameAndColorColumn = <T extends DomainObjectWithRequiredFields>(
   width: 350,
   accessor: (rowData: T) => rowData.otherPartyName,
   key: 'otherPartyName',
-  Cell: ({ rowData, column }) => {
-    const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
-
-    const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
-      setAnchorEl(event.currentTarget);
-    };
-
-    const handleClose = () => {
-      setAnchorEl(null);
-    };
-
-    return (
-      <Box
-        sx={{
-          flexDirection: 'row',
-          borderBottom: 'none',
-          alignItems: 'center',
-          display: 'flex',
-        }}
-      >
-        <ColorMenu
-          onClose={handleClose}
-          anchorEl={anchorEl}
-          onClick={color => {
-            handleClose();
-            onChange(rowData, color);
-          }}
-        />
-        <IconButton
-          labelKey="button.select-a-color"
-          icon={
-            <CircleIcon
-              htmlColor={rowData.color}
-              sx={{
-                width: '12px',
-                margin: 'margin: 0 9px 0 10px',
-                overflow: 'visible',
-                cursor: 'pointer',
-              }}
-            />
-          }
-          onClick={e => {
-            e.stopPropagation();
-            handleClick(e);
-          }}
-        />
-        <Box ml={1} />
-        {column.accessor(rowData)}
-      </Box>
-    );
-  },
+  Cell: ({ rowData, column }) => (
```

Comment:
I changed the color using this one and got 
```
Error
Unknown type "InvoicePatch". Did you mean "InvoiceNode"?:
```
which it shouldn't be either. I've been meaning to look at mutations.. maybe it is time

Reply:
Hmm true. I will tackle the mutations now that I can get the schema from the backend development branch